### PR TITLE
updating workflows - lts

### PR DESCRIPTION
**Pull request by ATorrise**
Action | Description
-- | --									
#6 | [update all workflows to remove deprecation warnings](https://github.com/zowe/zowe-cli-web-help-generator/pull/35/files), [octorelease updates](https://github.com/zowe/zowe-cli/pull/1712)										
#7 | remove Node 14 from workflow matrix

### CODE REVIEW

File: .github/workflows/changelog.yml
Comment (zFernand0) [inline .github/workflows/changelog.yml:1]: Could we change the `awharn/check_changelog_action@v0.0.2` to v1 here as well?

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -30,10 +30,10 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v1
```
Comment (zFernand0) [inline .github/workflows/codeql.yml:36]: thanks for changing the `init`
could you also update the `analyze`? 😋 
https://github.com/github/codeql-action#usage

**Review by zFernand0 — CHANGES_REQUESTED** [review]
LGTM! 😋 
just 2 small things to update 😅

File: .github/workflows/changelog.yml [hunk 2]
```diff
@@ -14,12 +14,12 @@ jobs:
...
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
         fetch-depth: 0
 
     - name: Check Changelog Updated
-      uses: awharn/check_changelog_action@v0.0.2
+      uses: awharn/check_changelog_action@1
```
Comment (t1m0thyj) [inline .github/workflows/changelog.yml:22]: This should be `@v1`

File: .github/workflows/zowe-cli-plugin.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3s
```
Comment (t1m0thyj) [inline .github/workflows/zowe-cli-plugin.yml:31]: This should be `@v3`

**Review by t1m0thyj — CHANGES_REQUESTED** [review]

**Review by zFernand0 — APPROVED** [review]

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**